packages/core/src/MapIt/index.js

```diff
@@ -42,6 +42,7 @@ function MapIt({
   codeType,
   indexColor,
   geoIndexMapping,
+  vulnerabilityLabel,
```

Comment:
I believe we call the value "score". Shouldn't this prop be called `scoreLabel` then?